doc/sphinx/addendum/type-classes.rst

```diff
@@ -564,12 +559,12 @@ Settings
    This flag allows to switch the behavior of instance declarations made through
    the Instance command.
 
-   + When it is on (the default), instances that have unsolved holes in
+   + When it is off (the default), they fail with an error instead.
```

Comment:
The commit fixing this cannot be backported (since it is not the same in master and v8.9) so this should be moved to another PR. Sorry about that :disappointed:

Reply:
Removed